Helper/Cart.php

```diff
@@ -1702,6 +1700,20 @@ public function buildCartFromQuote($quote, $immutableQuote, $items, $placeOrderP
         //Store immutable quote id in metadata of cart
         $cart['metadata']['immutable_quote_id'] = $immutableQuote->getId();
 
+        // Transmit session ID via cart metadata, making it available for session emulation in API calls
+        if ($this->deciderHelper->isAddSessionIdToCartMetadata()) {
```

Comment:
Just to double check. If it ok that only first expression is under feature switch?

Reply:
The other expression is for the case when we need a reliable way to transmit some 3rd party (session) data so we can reset / use it in API call session, for instance ID.me data mentioned in the task (uuid, group, subgroups). It's not commonly used, only when it's required by the supported plugins (ID.me in this case). It is a new thing, not supported previously.

Reply:
So it will work even without feature switch if merchant doesn't use load balancer. Correct?

Reply:
Correct.

Reply:
Got it, thank you